package.json

```diff
@@ -2,48 +2,54 @@
   "name": "sauce-testcafe-runner",
   "version": "0.0.0",
   "description": "Sauce Labs Testcafe runner",
-  "author": "<dev@saucelabs.com>",
+  "author": "<devx@saucelabs.com>",
   "license": "MIT",
   "private": true,
   "bin": "./bin/testcafe",
-  "main": "src/console-wrapper.js",
+  "main": "lib/console-wrapper.js",
   "scripts": {
-    "test": "node src/console-wrapper.js",
-    "lint": "eslint tests/ src/",
+    "build": "tsc",
+    "test": "tsc; node lib/console-wrapper.js",
+    "lint": "eslint tests/ src/**/*.ts",
     "unit-test": "jest --env node"
   },
   "keywords": [
     "testcafe"
   ],
   "dependencies": {
     "@saucelabs/testcomposer": "^1.0.0",
+    "@tsconfig/node18": "^1.0.1",
```

Comment:
This on the other hand should be a devDependency, no?

Reply:
I think it needs to be in `dependencies`. I tried with `devDependencies` but it cannot find the extended tsconfig. https://github.com/saucelabs/sauce-testcafe-runner/actions/runs/4727388391/jobs/8387951231?pr=175#step:6:350

Reply:
I'm late to the party, but just wanted to say, the reason it needed to be in `dependencies` and not in `devDependencies` is because the Dockerfile is running npm ci with `--omit=dev`: https://github.com/saucelabs/sauce-testcafe-runner/actions/runs/4727388391/jobs/8387951231?pr=175#step:6:321

It should really be in devDependencies